### PR TITLE
Update value.md

### PR DESCRIPTION
**Pull request by CODE-REaD**
I see no reason to include an additional `prop.resolve( count );` call here, suggest removing as it adds confusion.

### CODE REVIEW

File: docs/value.md [hunk 1]
```diff
@@ -27,7 +27,6 @@ Specify the behavior of a property by listening to changes in other properties.
         prop.listenTo( "name", () => {
           prop.resolve( ++count );
         } );
-        prop.resolve( count );
```
Comment (chasenlehara) [inline docs/value.md]: Without this line, `nameChangeCount` would start off as `undefined`, right?

I think this line is required so `nameChangeCount` starts off as `0`.

Comment (CODE-REaD) [reply docs/value.md]: I just tested and to my surprise **you are correct**, though as far as the example goes line 30 doesn't add anything.  In fact, it appears to me that line 30 _**only**_ gets called when `nameChangeCount` is initially referenced _directly_ (not as a result of a `listenTo()`.  That's not what I would expect reading this as JavaScript.  Also it seems line 30 only gets called in case `nameChangeCount` isn't yet defined.  Again, not what I would expect (see my tests at https://codepen.io/codeREaD/pen/LXxMLN?editors=0011).

Bottom line: technically line 30 should stay, but I wish there were a more straightforward way to initialize `nameChangeCount`.

Comment (CODE-REaD) [reply docs/value.md]: I  agree that `let count = prop.resolve(0)` is not better!

Sorry to belabor this, but part of my confusion is that `name` is _not_ defined by default, so is it not "changed" by the `p.name = "Justin"` call.  I would describe it rather as having been "set."

By that logic, `nameChangeCount` _should_ start off as undefined, because no changing or setting has actually occurred yet (and if you're still with me it would be more clear to label this value `nameSetCount` and omit line 30.)

If you want I will cancel this pull request and submit another one with all these changes.

Comment (phillipskevin) [reply docs/value.md]: `p.name = "Justin"` changes `name` from `undefined` to `Justin`.

And I do not think a "count" should be undefined... it should always be a number.

Comment (chasenlehara) [reply docs/value.md]: @CODE-REaD You can think of `name` changing from `undefined` to `"Justin"` with the first call; setting the property is changing the value. Re: `nameChangeCount`, if the name hasn’t changed, the count should be 0. For that reason, I’m going to close this PR.

You’re right that there is a difference between something being changed vs. set… and in this example, we’re counting how many times the property has changed, not how many times it has been set. If you set the property to the same value it already had, no change event is emitted. I think `nameChangeCount` is a good name for that property, so I wouldn’t open a PR to change the property names.
